eZ/Publish/Core/Repository/Values/ObjectState/ObjectState.php

```diff
@@ -26,16 +23,33 @@
  */
 class ObjectState extends APIObjectState
 {
-    use MultiLanguageTrait;
-    use MultiLanguageNameTrait;
-    use MultiLanguageDescriptionTrait;
```

Comment:
What about instead deprecating property 'defaultLanguageCode', making it available via magic __get() for BC, and switch over to using 'mainLanguageCode' even for obj states and stategroups ?
Would it make more sense?